train.py

```diff
@@ -109,6 +118,10 @@ def main(args, init_distributed=False):
     train_meter.stop()
     print('| done training in {:.1f} seconds'.format(train_meter.sum))
 
+    if logging:
+        experiment.log_metrics({'valid_loss': valid_losses[0]})
```

Comment:
Could you please change this to 'val_loss' and add 'lr' together with other metrics in the valid_loss array?